src/disaster_vehicle.cpp

```diff
@@ -384,13 +384,13 @@ static bool DisasterTick_Ufo(DisasterVehicle *v)
 		if (z <= u->z_pos && (u->vehstatus & VS_HIDDEN) == 0) {
 			v->age++;
 			if (u->crashed_ctr == 0) {
-				u->Crash();
+				uint pass = u->Crash();
```

Comment:
It might be better if it were to be just `victims`, as it also includes the driver.
The same holds for the other uses of `pass` and/or `num` in this PR.

Reply:
Variables renamed where appropriate.